src/components/react-virtualized/AutoScroll.js

```diff
@@ -71,10 +69,13 @@ export default class AutoScroll extends PureComponent {
       return
     }
 
-    // We way need to auto scroll when:
-    // - Rows has been changed.
-    // - Parent size has changed.
-    if (rowsHasChanged || nextProps.height !== height) {
+    // We way need to auto scroll to the bottom when:
+    // - When a new message was received.
+    // - Parent size has changed and the scroll position was close to the bottom.
+    if (
+      (nextProps.receivedMessageViaSocket && rowsHasChanged) ||
```

Comment:
No, it will only scroll to the bottom if you in the endThreshold. This behaviour was the desired one by Carina as explained here: https://uebergrape.staging.chatgrape.com/chat/channel/632:94f51c6c8f1e11e8bd180242ac190002/